game.libretro.picodrive/addon.xml.in

```diff
@@ -1,10 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <addon id="game.libretro.picodrive"
 		name="Sega - MS/GG/MD/CD/32X (PicoDrive)"
-		version="2.00.0.40"
+		version="2.00.0.42"
```

Comment:
Can you revert? The CI was handling one version bump just fine but was breaking on two version bumps.

Reply:
I've added a third commit to fix the version number. Is it ok? In the future, should I avoid bumping the version completely, or is the bump by 1 ok?

Reply:
Avoid bumping the version, best to let the CI script take care of it.